dotnet/src/SemanticKernel.Abstractions/AI/TextCompletion/CompleteRequestSettings.cs

```diff
@@ -81,5 +81,12 @@ public static CompleteRequestSettings FromCompletionConfig(PromptTemplateConfig.
             MaxTokens = config.MaxTokens,
             StopSequences = config.StopSequences,
         };
+
+        if (config.ChatSystemPrompt is not null)
```

Comment:
What if the `ChatSystemPrompt` property is empty? should it be added?

Reply:
Good point, going to change it to `string.IsNullOrWhitespace()`